avidemux_sk.ts

```diff
@@ -10726,47 +10728,47 @@ Only VP8/VP9/AV1 video and Vorbis/Opus audio supported</source>
     <message>
         <location line="+56"/>
         <source>Average Bitrate:</source>
-        <translation type="unfinished"></translation>
+        <translation>Priemerná prenosová rýchlosť:</translation>
     </message>
     <message>
         <location line="+35"/>
         <source>Pixel format:</source>
-        <translation type="unfinished"></translation>
+        <translation>_Formát pixelov:</translation>
```

Comment:
Please fix the leading underscore. (Was it a bad suggestion by translation software? There is a different location in common code where we indeed use an accelerator as underscore, but here we are in Qt code where 1) an accelerator is `&amp;` and 2) we don't need any in this case.)